src/core/js/models/adaptModel.js

```diff
@@ -739,8 +747,20 @@ define([
 
         onIsComplete: function() {
             this.checkCompletionStatus();
-
             this.checkLocking();
+        },
+
+        onAll: function(type, model, value) {
```

Comment:
a JSDoc comment would be useful here to document the function parameter data types at least